logos/benchmarks/benchmark_logos.py

```
@@ -1248,6 +1284,494 @@ def generate_charts(out_dir: Path, results: list[RequestResult], tracker, t0: fl
     print(f"  [charts] Written to {out_dir}")
 
 
+# ── Service management ────────────────────────────────────────────────────
+
+
+def _logos_is_running(url: str, logos_key: Optional[str] = None) -> bool:
+    """Return True if the Logos /v1/models endpoint responds with 2xx."""
+    try:
+        headers = {"logos_key": logos_key} if logos_key else {}
+        r = httpx.get(f"{url.rstrip('/')}/v1/models", headers=headers, timeout=5.0, verify=False)
```

Ruff (0.15.15)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)